.storybook/main.js

```diff
@@ -1,7 +1,7 @@
 const path = require("path");
 
 module.exports = {
-  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  stories: ["../src/Button/Button.stories.js", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
```

Comment:
You shouldnt need to add button here. It should pick up the button story as long as you have .stories.js.. Thats what `"../src/**/*.stories.@(js|jsx|ts|tsx)"]` is